src/types/reduxActions.js

```diff
@@ -243,6 +243,7 @@ export type Action =
         }
       }
     }
+  | { type: 'FIO/FIO_WALLET_BY_ADDRESS', data: { wallet: EdgeCurrencyWallet | null } }
```

Comment:
Unnecessary redux. Please remove and factor

Reply:
refactored